Sources/Sapi.php

```diff
@@ -72,14 +72,11 @@ class Sapi
 	 */
 	public static function isSoftware(string|array $server): bool
 	{
-		$servers = (array) $server;
-
-		foreach ($servers as $server) {
-			if (
-				isset($_SERVER['SERVER_SOFTWARE'])
-				&& strpos($_SERVER['SERVER_SOFTWARE'], self::$server_software[$server] ?? $server) !== false
-			) {
-				return true;
+		if (isset($_SERVER['SERVER_SOFTWARE'])) {
+			foreach ((array) $server as $server) {
```

Comment:
Won't PHP hate that and overwrite `$server`?

Reply:
Derp. Good catch.

Reply:
Fixed